subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync.c

```diff
@@ -921,7 +921,7 @@ static void isr_rx_done_cleanup(struct lll_sync *lll, uint8_t crc_ok, bool sync_
 static void isr_done(void *param)
 {
 	lll_isr_status_reset();
-	isr_rx_done_cleanup(param, ((trx_cnt > 1U) ? 1U : 0U), false);
+	isr_rx_done_cleanup(param, ((trx_cnt) ? 1U : 0U), false);
```

Comment:
I'm curious, why is the `crc_ok` parameter for `isr_rx_done_cleanup` not a `bool`?

Reply:
`crc_ok` is assigned to 1-bit `crc_valid` in the extra type structure and passed to ULL.

Reply:
True, but then then in `isr_rx_done_cleanup` we could have 

```
e->crc_valid = crc_ok ? 1U : 0;
```
instead of the current
```
e->crc_valid = crc_ok;
```

So that the conversion to the 1-bit value happens as late as possible, and would doing the `? 1U : 0` as often. 
At least to me it makes more sense to use the `bool` type for boolean values where possible, rather than `uint8_t` (which is often used to store boolean rules in at the moment). 

In any case, this is outside the scope of this PR, and no changes are needed.